guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
<p>"and its paper has more than 1500 citations." -&gt; this number can change quickly. Perhaps we can just say "its paper is one of the most cited papers in the field".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Hi, changed it to "its paper is highly cited by the computer vision community"</p><p> </p>